src/jenkins_job_insight/main.py

```
@@ -4464,6 +4464,79 @@ async def unsubscribe_notifications(body: UnsubscribeRequest, request: Request):
+@app.get("/api/users/mentions")
+async def get_user_mentions(request: Request):
+    """Get comments that mention the current user."""
+    username = request.state.username
+    if not username:
+        raise HTTPException(status_code=401, detail="Username required")
+    _check_allow_list(request)
+    try:
+        offset = max(0, int(request.query_params.get("offset", "0")))
+        limit = min(200, max(1, int(request.query_params.get("limit", "50"))))
+    except ValueError:
+        raise HTTPException(status_code=400, detail="offset and limit must be integers")
```

Ruff (0.15.11)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)